docker-setup/snowflake-kafka-docker/kafka-connect.yml

```diff
@@ -65,10 +65,7 @@ services:
       - 9876:9876 # JMX port
       - 8085:8085
     volumes:
-      - /usr/local/share/kafka/plugins:/etc/kafka-connect/jars/ # if you want your custom jar, put custom jar in ./jars directory
-    #  - ./prometheus:/opt/jmx-exporter/
-    # volumes:
-    #   - ./config/connect-log4j.properties:/etc/kafka/connect-log4j.properties
```

Comment:
should we keep it here? I assume they're needed in some cases

Reply:
They may be needed in non m1 environments, however I haven't needed these. I think its ok to keep it here

Reply:
I havent had a successful run with prometheus and grafanna. 
Can you instead keep it here (Commented) and make a comment here saying we dont need this ATM. Only needed for exporting JMX to prometheus